mobile/native/src/ln_dlc/mod.rs

```diff
@@ -89,12 +90,32 @@ pub const FUNDING_TX_WEIGHT_ESTIMATE: u64 = 220;
 static NODE: Storage<Arc<Node>> = Storage::new();
 static SEED: Storage<Bip39Seed> = Storage::new();
 
+/// Trigger an on-chain sync followed by an update to the wallet balance and history.
+///
+/// We do not wait for the triggered task to finish, because the effect will be reflected
+/// asynchronously on the UI.
 pub async fn refresh_wallet_info() -> Result<()> {
     let node = NODE.try_get().context("failed to get ln dlc node")?;
     let wallet = node.inner.wallet();
 
-    spawn_blocking(move || wallet.sync()).await??;
-    keep_wallet_balance_and_history_up_to_date(node)?;
```

Comment:
🤔 Maybe it will be faster if we run `keep_wallet_balance_and_history_up_to_date` before everything else? It may use outdated data, but that should be updated after the sync immediately.

Reply:
I considered that too, but now this is only called if the user pulls down on the wallet screen. In such a scenario, I think it would be okay to wait for the syncs before updating the UI.

On startup this is no longer called.